### ISSUE TITLE
About the Figure 4 in the paper

### ISSUE THREAD

**Issue by MAX-OTW** [valeriopaolicelli/adageo-WACV2021#3]
Hello, @gmberton ,I really appreciate for your work.
The Figure 4:(Visualization of attention score maps）in the paper looks good, currently I want to try to implement it but I failed. Is it convenient to provide the relevant implementation code?Of course, detailed guidance would be better.
I am always looking forward to your kind response.
Best regards.

**Comment by gmberton** [comment]
Hello @MAX-OTW, the attention map was implemented by @valeriopaolicelli, which does not work in my lab anymore. You can try to contact him to have some info about how to implement it.

**Comment by valeriopaolicelli** [comment]
Hello @MAX-OTW , 
sorry for the late reply, but I’m no longer supporting and monitoring this project. 
Unfortunately I don't have access to the original resources used for this work. Nevertheless, I found something used with the latest release of AdaGeo ([AdaGeo-lite](https://www.frontiersin.org/articles/10.3389/fcomp.2022.841817/full)). I've uploaded a draft [code](https://github.com/valeriopaolicelli/adageo-pytorch/blob/main/src/extract_cam.py) to extract CAM heatmaps using the ResNet18 backbone and one of our checkpoints. I am not able to test this code snippet, but I hope it can help you.

BR
Valerio
